WordPress/Classes/Services/SharingService.swift

```diff
@@ -61,6 +61,7 @@ open class SharingService: LocalCoreDataService {
     @objc open func syncPublicizeConnectionsForBlog(_ blog: Blog, success: (() -> Void)?, failure: ((NSError?) -> Void)?) {
         let blogObjectID = blog.objectID
         guard let remote = remoteForBlog(blog) else {
+            failure?(nil)
```

Comment:
IMHO `success()` and `failure(error)` make sense, whereas `failure(nil)` feels a bit unecessarily ambiguous.

Considering we know exactly what happened it may be a good idea to return an error object here.

Reply:
Thanks for the note! I've added an error, which is now logged out when syncing blog details:

```
2021-05-17 11:44:39:942 WordPress[52465:2163375] Failed syncing publicize connections for blog https://example.wordpress.com: WordPress.SharingService.SharingServiceError.siteWithNoRemote
```